runtime/doc/pattern.txt

```diff
@@ -170,6 +170,16 @@ found", for Vi compatibility.
 For the |:s| command the "e" flag can be used to avoid the error message
 |:s_flags|.
 
+					*search-options*
+The following options affect how a search is performed in Vim:
+    'hlsearch'		highlight matches
+    'ignorecase'	ignore case when searching
```

Comment:
Ideally these would have bullet points, but I guess we don't own pattern.txt so won't be able to change to to "flow layout" anytime soon.